src/network/types.ts

```diff
@@ -38,12 +38,12 @@ type ProxyConfig = {
 };
 
 type ConnectionInfo = {
-  nodeId: NodeId;
-  certificates: Array<Certificate>;
-  egressHost: Host;
-  egressPort: Port;
-  ingressHost: Host;
-  ingressPort: Port;
+  remoteNodeId: NodeId;
+  remoteCertificates: Array<Certificate>;
+  localHost: Host;
+  localPort: Port;
+  remoteHost: Host;
+  remotePort: Port;
```

Comment:
Both `local` and `remote` are the `proxyHost` and `proxyPort` for local and remote proxies.

Given that the names no longer match what we have in the rest of the codebase, you'd want to add a comment above about that.

Reply:
Sure, I'll add a comment to the type for that.